Cannot find module '@nestjs/swagger/dist/interfaces/open-api-spec.interface' or its corresponding type declarations.

**Describe the bug**
Using the following library version:
nestjs-zod: 5.2.1 (latest)
@nestjs/swagger: 11.2.6 (latest)

When I want to build my application (using tsc), I’m having the following error:
```
node_modules/nestjs-zod/dist/index.d.mts:9:30 - error TS2307: Cannot find module '@nestjs/swagger/dist/interfaces/open-api-spec.interface' or its corresponding type declarations.

9 import { SchemaObject } from "@nestjs/swagger/dist/interfaces/open-api-spec.interface";
```
Found 1 error in node_modules/nestjs-zod/dist/index.d.mts:9

Here is my tsconfig file (ESM project):
```
{
    "compilerOptions": {
        "target": "ES2023",
        "module": "nodenext",
        "moduleResolution": "nodenext",
        "moduleDetection": "force",
        "resolveJsonModule": true,
        "allowJs": true,
        "esModuleInterop": true,
        "isolatedModules": true,
        "emitDecoratorMetadata": true,
        "experimentalDecorators": true,
        "strict": true,
        "outDir": "dist",
        "types": ["node"],
        "baseUrl": ".",
        "rootDir": "src",
    },
    "include": ["src"]
}

```

**Expected behavior**
The error should not occur and the build should work

**Actual behavior**
Error occurred while transpiling (see error on top)

--- a/packages/nestjs-zod/src/zodV3ToOpenApi.ts
+++ b/packages/nestjs-zod/src/zodV3ToOpenApi.ts
@@ -4,10 +4,13 @@ import { z } from 'zod/v3';
 import deepmerge from 'deepmerge';
 import type { OpenAPIObject } from '@nestjs/swagger';
 
-type SchemaObject = Exclude<Exclude<
-  Exclude<OpenAPIObject['components'], undefined>['schemas'],
-  undefined
->[string], { $ref: string }>
+type SchemaObject = Exclude<
+  Exclude<
+    Exclude<OpenAPIObject['components'], undefined>['schemas'],
+    undefined
+  >[string],
+  { $ref: string }
+>;
 
 export interface ExtendedSchemaObject extends SchemaObject {
   [key: `x-${string}`]: any;

